saleor/account/models.py

```diff
@@ -282,3 +282,27 @@ class Meta:
 
     def __repr__(self):
         return f"{self.__class__.__name__}(type={self.type!r}, user={self.user!r})"
+
+
+class StaffNotificationRecipient(models.Model):
+    user = models.OneToOneField(
+        User,
+        related_name="staff_notification",
+        on_delete=models.CASCADE,
+        blank=True,
+        null=True,
+    )
+    staff_email = models.EmailField(unique=True, blank=True, null=True)
+    site_settings = models.ForeignKey(
```

Comment:
I think that we actually don't need this relationship. I thought we will use `SiteSettings` model to store these addresses but since it's a separate model, is there any use case for that?

Reply:
I don't think so